src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/repository.py

```diff
@@ -25,6 +26,8 @@ def _obtain_data_from_registry(login_server, path, resultIndex, username, passwo
             )
         )
 
+        log_request(None, response.request)
```

Comment:
Please correct me if I'm wrong. According to this: http://msrest.readthedocs.io/en/latest/_modules/msrest/http_logger.html. The `log_request` will log all the headers, including Authorization header. Won't this be a security concern?